backend/db/utils.py

```diff
@@ -271,9 +271,9 @@ def get_objs_by_composite_key(con, table: str, keys: List[str], objs: List[Dict]
     todo: refactor: look to insert_from_dicts() more secure way to do this by passing `params` to run_sql()`
     :return: dictionary with keys as the primary key and values as the row contents"""
     key_vals = {key: [str(obj[key]) for obj in objs] for key in keys}
+    conditions = ' AND '.join([f'"{k}" {sql_in(v, True)}' for k, v in key_vals.items()])
```

Comment:
### Quoting of keywords used in column names (`primary_key`, `table`, etc)
@Sigfried I made this change for `get_obj_by_composite_key()` and `get_objs_by_composite_key()` for the purposes of this PR, but this doesn't give us full coverage. I guess theoretically we should always be quoting the column names for these generalized functions.